src/main/java/bio/terra/stairway/Stairway.java

```diff
@@ -189,6 +201,37 @@ public Stairway(Stairway.Builder builder) {
     this.projectId = builder.projectId;
     this.workQueueEnabled = (projectId != null);
     this.quietingDown = new AtomicBoolean();
+    this.stairwayHook =
+        (builder.stairwayHook == null) ? populateDefaultStairwayHook() : builder.stairwayHook;
+  }
+
+  public StairwayHook populateDefaultStairwayHook() {
```

Comment:
Again, organization - not sure if this is the best place for this chunk of code.

Reply:
Based on other code I looked at, this seems fine.